modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/dto/v1_0/converter/ObjectEntryDTOConverter.java

```diff
@@ -128,6 +125,43 @@ private DTOConverterContext _getDTOConverterContext(
 			dtoConverterContext.getUser());
 	}
 
+	private Object[] _getManyToManyRelationshipObjectEntries(
+		DTOConverterContext dtoConverterContext, int nestedFieldsDepth,
+		ObjectRelationship objectRelationship,
+		com.liferay.object.model.ObjectEntry objectEntry) {
+
+		try {
+			boolean reverse = objectRelationship.isReverse();
+
+			if (objectRelationship.isReverse()) {
+				objectRelationship =
+					_objectRelationshipLocalService.
+						fetchReverseObjectRelationship(
+							objectRelationship, false);
+			}
+
+			Pagination pagination = Pagination.of(1, 20);
```

Comment:
Are we always returning the first 20 relationships?

Reply:
Already answered